_data/strings.yml

```
@@ -14,6 +14,13 @@ locales:
     made_possible: "Dieses Projekt wird von den <a href='https://github.com/qmk'>QMK-Entwicklern</a> und Mitwirkenden wie Ihnen unterstützt!"
     hosted_on: "Betrieben mit GitHub Pages &mdash; Layout von <a href='https://github.com/orderedlist'>orderedlist</a>"
     description: "Eine Open Source Firmware für auf AVR oder ARM basierende Tastaturen"
+  en: &DEFAULT_MR
```

This needs to be the language code for Marathi.

```suggestion
  mr: &DEFAULT_MR
```